cypress/integration/Campaign Management/campaign_enrollment_and_entry_creation.js

```diff
@@ -51,6 +53,31 @@ describe('Campaign Enrollment Test Suite', () => {
             })
     })
 
+    it('Create Campaign Entry', () => {
+        cy.login({id: 2})
+
+        cy.intercept('GET', campaign_exploration_link).as('campaign_detail')
+        cy.visit(campaign_exploration_link)
+        cy.wait('@campaign_detail')
+
+        cy.contains('button', 'Entry')
+            .click()
+            .then(() => {
+                cy.get('#levels').clear().type(levelup)
+                cy.contains('button', 'Continue').click()
+                cy.contains('button', 'Create').click()
+                cy.url().should('include', campaign_exploration_id)
+            })
+            .then(() => {
```

Comment:
is there a reason why we are chaining `.then()` here? all of these are sequential events that do not depend on each other

Reply:
The `.then()` segment  will only succeed if the entry is created in the lines right before. Otherwise campaign detail will have no entries.

Reply:
ok